src/types.jl

```diff
@@ -551,7 +559,7 @@ for (C, acol, cola) in [(DIN99d, :ADIN99d, :DIN99dA),
     cfn = Expr(:tuple, colorfields(C)...)
     elty = eltype_default(C)
     ub   = eltype_ub(C)
-    Csym = C.name.name
+    Csym = isdefined(Core, :UnionAll) ? Base.datatype_name(C) : C.name.name
```

Comment:
Since ColorTypes was already completely hosed on 0.6 before this, and it hosed tests for ~50 packages, I don't think we need to wait to resolve this before merging https://github.com/JuliaLang/METADATA.jl/pull/7899.